library/core/benches/fmt.rs

```diff
@@ -1,13 +1,13 @@
 use std::fmt::{self, Write as FmtWrite};
 use std::io::{self, Write as IoWrite};
-use test::Bencher;
+use test::{black_box, Bencher};
```

Comment:
std has [hint::black_box](https://doc.rust-lang.org/stable/std/hint/fn.black_box.html), that can be used directly.

Reply:
yes, it essentially the same https://github.com/rust-lang/rust/blob/62cfd8a123e2ee78fab6ca35ae8bffded28ab620/library/test/src/bench.rs#L22

seems all the code in library/bench use `test::black_box` insead of `use std::hint::black_box`, so we'd better keep the same here?